canonical/ubuntu.com
# add cred maintenance window

## Done

- Add a maintenance window starting from 28 Sep 2024 11AM UTC to 28 Sep 2024 3PM UTC

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure banner shows up for maintenance window
- Shop should not be accessible anymore
- No exams should be scheduled within the maintenance window

## Issue / Card

Fixes [#15341](https://warthogs.atlassian.net/browse/WD-15341)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
